keyserver/handler.go

```diff
@@ -0,0 +1,15 @@
+package keyserver
+
+import . "github.com/coniks-sys/coniks-go/protocol"
+
+func (server *ConiksServer) handleRegistration(reg *RegistrationRequest) (Response, error) {
+	if len(reg.Username) == 0 || len(reg.Key) == 0 {
```

Comment:
Sorry I missed this when I was checking through the comments earlier (so many comments...). It seems you decided to get rid of `handler.go`, which I think is fine. The only server-related tasks I can think of right now that we may need to perform in the handlers are possibly reading from the DB. But even those, I think, may be abstracted away in the directory, no?